common/src/main/java/org/apache/rocketmq/schema/registry/common/storage/StorageService.java

```diff
@@ -95,4 +97,8 @@ default List<String> listSubjectsByTenant(StorageServiceContext storageServiceCo
     default List<String> listTenants(StorageServiceContext storageService, QualifiedName name) {
         throw new UnsupportedOperationException(ERROR_MESSAGE_DEFAULT);
     }
+
+    @Override
+    default void close() throws IOException {
+    }
```

Comment:
I think the default keywords before all method in this class is unnecessary. can you remove them? DefaultStorageServiceImpl class is never used, and can be removed.

Reply:
> DefaultStorageServiceImpl

JdbcStorageService class is in use